genesis_ledgers/phase_three/annotated_ledger.json

```diff
@@ -1,2998 +1,212 @@
-   "sk": null,
-   "balance": "0.000000000",
-   "delegate": null,
-   "discord_username": "CodaBP0"
-  },
-  {
-   "pk": "4vsRCVnNSTUkMmWBNeDawMwHUsQsNdjeMVgAfFyfnxedPqdk2GNnauhqGp9pkp4N5Y9R6UEk2ixUw4mXRzBLVYAMisDBqnfT7kn3gUeccZGMDyF2K2q2e3kbktQfmsqvNTNGicFtXaB9W6TW",
-   "sk": null,
-   "balance": "0.000720000",
-   "delegate": "4vsRCVQAkUfP9wGLx8yiKv3GPVATRBFW4kuzj2XA8tihDjgfmNFNHZNjWsynvHNJbTQcr8atGquTieHAQtD3j9odMiS9ApZqKcFTzHggYwP4haXm6LgsBXei5vgDCFa88ZRo9Kzi9sHFRTkK",
-   "delegate_discord_username": "CodaBP1"
-  },
-  {
-   "pk": "4vsRCVQAkUfP9wGLx8yiKv3GPVATRBFW4kuzj2XA8tihDjgfmNFNHZNjWsynvHNJbTQcr8atGquTieHAQtD3j9odMiS9ApZqKcFTzHggYwP4haXm6LgsBXei5vgDCFa88ZRo9Kzi9sHFRTkK",
-   "sk": null,
-   "balance": "0.000000000",
-   "delegate": null,
-   "discord_username": "CodaBP1"
-  },
-  {
-   "pk": "4vsRCVbuUEjej7M8MnxVNB9PzaXbWciy5zNb8iNQ3X24yafxYAik3iSUd6n89Z47VciTbFZMdDG5wYypBukACJ1f6Aduvr8FxFzecA1UvmVbcpzmycJbnU1wYQ3CJpYTa8SDrMT36NRNhqC8",
-   "sk": null,
-   "balance": "0.000720000",
-   "delegate": "4vsRCVNep7JaFhtySu6vZCjnArvoAhkRscTy5TQsGTsKM4tJcYVc3uNUMRxQZAwVzSvkHDGWBmvhFpmCeiPASGnByXqvKzmHt4aR5uAWAQf3kqhwDJ2ZY3Hw4Dzo6awnJkxY338GEp12LE4x",
-   "delegate_discord_username": "CodaBP2"
-  },
-  {
-   "pk": "4vsRCVNep7JaFhtySu6vZCjnArvoAhkRscTy5TQsGTsKM4tJcYVc3uNUMRxQZAwVzSvkHDGWBmvhFpmCeiPASGnByXqvKzmHt4aR5uAWAQf3kqhwDJ2ZY3Hw4Dzo6awnJkxY338GEp12LE4x",
-   "sk": null,
-   "balance": "0.000000000",
-   "delegate": null,
-   "discord_username": "CodaBP2"
-  },
-  {
-   "pk": "4vsRCVf5KvBNPwhm9tyLq6KdN7uB9myHRa1bzMDCQRAXjHbib9weCATBaHsq9UP4UAETnYAoCrQrUVRu53dZMo5F8bmXj6oRxwwa3bdtcJZRyKvPST5tdz1n1ZvX2Qkb5fUCzsiixjBnNAJb",
-   "sk": null,
-   "balance": "0.000720000",
-   "delegate": "4vsRCVeWLdYZSH4VKZrusQMrthSU1zyTEhJcSBtX4THNRXV1oTB1iBbNwPGBVWqPxC1QQ48R5X9iWjB8ZnAgX1FkFafiZqaUSqZtZxjNNaWfd7VgFFHQCV2Aukd9DYRDRT25nsQF61ajFnPk",
-   "delegate_discord_username": "CodaBP3"
-  },
-  {
-   "pk": "4vsRCVeWLdYZSH4VKZrusQMrthSU1zyTEhJcSBtX4THNRXV1oTB1iBbNwPGBVWqPxC1QQ48R5X9iWjB8ZnAgX1FkFafiZqaUSqZtZxjNNaWfd7VgFFHQCV2Aukd9DYRDRT25nsQF61ajFnPk",
-   "sk": null,
-   "balance": "0.000000000",
-   "delegate": null,
-   "discord_username": "CodaBP3"
-  },
-  {
-   "pk": "4vsRCVRamY8S6BDdRQtsjy4urAqztxTD2aZyvN6T8cjs8Y3wSZd8GtLv9eFp7jrJ1dBd6WG7H8KN3Tk7aLtpBjvHEUAmHnt8MGPzfj2MW3eXPUgccy9ed5UFdccwYZYxYpbyXyDkDdeXsnxj",
-   "sk": null,
-   "balance": "0.000720000",
-   "delegate": "4vsRCVmwWh3SviHCowUvxRNM7qub9KWowwkcTLteBJYQLL3BCmnrHqYfTjrMYi6w6F88Te81nrVjuNtXTa84ppK62SCwNh1rVKwuqoaiJd881PaDWyBoBQc5G8hpXxzACixDHmnh8755v2zw",
-   "delegate_discord_username": "CodaBP4"
-  },
-  {
-   "pk": "4vsRCVmwWh3SviHCowUvxRNM7qub9KWowwkcTLteBJYQLL3BCmnrHqYfTjrMYi6w6F88Te81nrVjuNtXTa84ppK62SCwNh1rVKwuqoaiJd881PaDWyBoBQc5G8hpXxzACixDHmnh8755v2zw",
-   "sk": null,
-   "balance": "0.000000000",
-   "delegate": null,
-   "discord_username": "CodaBP4"
-  }
- ]
+ {
```

Comment:
Not sure this should be removed as part of this PR

Reply:
These are not. I added them to run a small qa-net before landing this on develop. Will revert after validating the qa-net